### PR TITLE
[RFC][UN-14942] Fix XSS vectors

### PR DESCRIPTION
**Pull request by migueldemoura**
https://uniplaces.atlassian.net/browse/UN-14942

**This is a breaking change**. We now also need to explicitly call `htmlSafe` for all the properties whose template changed from `{{{ prop }}}` to `{{ prop }}`.

### CODE REVIEW

File: addon/components/uni-carrousel.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import { gt } from '@ember/object/computed';
 import Component from '@ember/component';
 import { computed } from '@ember/object';
-import $ from 'jquery';
```
Comment (rntdrts) [inline addon/components/uni-carrousel.js:4]: 👍

**Review by rntdrts — APPROVED** [review]

**Review by mmelo — APPROVED** [review]

**Review by tlfbrito — APPROVED** [review]

File: tests/dummy/app/templates/components/demo-page.hbs [hunk 2]
```diff
@@ -2,7 +2,7 @@
 {{! template-lint-disable attribute-indentation }}
 
 <div class="demo-app__banner">
-  {{uni-title title="ember-cli-uniq" subtitle=(concat "made with ♥ at " (svg-jar "uniplaces_logo_white"))}}
+  {{uni-title title="ember-cli-uniq" subtitle="made with ♥ at Uniplaces"}}
```
Comment (AndreJoaquim) [inline tests/dummy/app/templates/components/demo-page.hbs:5]: Why?!

File: addon/components/uni-carrousel.js [hunk 3]
```diff
@@ -22,11 +21,11 @@ export default Component.extend(EKMixin, {
     return this.get('page') * this.get('componentWidth') * -1;
   }),
   wrapperStyle: computed('deltaX', function() {
-    return htmlSafe(`transform: translate3d(${this.get('deltaX')}px, 0px, 0)`);
+    return htmlSafe(`transform: translate3d(${parseFloat(this.deltaX)}px, 0px, 0)`);
```
Comment (AndreJoaquim) [inline addon/components/uni-carrousel.js:24]: This does more than you know... you just made this only compatible with ember 3.0 onwards by moving from `this.get('deltaX')` to `this.deltaX`

File: addon/components/uni-photo-thumbnail.js [hunk 4]
```diff
@@ -20,7 +21,7 @@ export default Component.extend({
   }),
 
   imageStyle: computed('photo.url', function() {
-    return htmlSafe(!this.get('photo.url') ? '' : `background-image: url(${this.get('photo.url')})`);
+    return htmlSafe(this.get('photo.url') ? `background-image: url(${encodeURI(this.get('photo.url'))})` : '');
```
Comment (AndreJoaquim) [inline addon/components/uni-photo-thumbnail.js:24]: There's no test that verifies this behaviour still works correctly...

File: addon/components/uni-carrousel.js (hunk 3 above)
Comment (migueldemoura) [reply addon/components/uni-carrousel.js:24]: Addressed in the branch. Nice catch!

**Pull request merged**